[io]write ut

liyigang1 opened the pull request:
1.添加fileoperations插件的ut
2.添加fileoperationseventhandler、fileoperationseventreciver、trashfileeventreciver、filecopymovejob、fileoperationsservice、operationsstackproxy等类的ut

liyigang1 committed:
fix: Write Ut

Write UT for asyncfileinfo and localdiriterator classes

Log: Write UT for asyncfileinfo and localdiriterator classes
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 committed:
feat: Write Ut

Write ut for the fileoperations class

Log: Write ut for the fileoperations class
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 committed:
feat: write ut

Write ut for the fileoperationseventhandler and fileoperationseventreceiver classes

Log: Write ut for the fileoperationseventhandler and fileoperationseventreceiver classes
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 committed:
feat: write ut

Improve the ut of the fileoperationsventreceiver class

Log: Improve the ut of the fileoperationsventreceiver class
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 committed:
fix: write ut

Write ut for the trashfileeventreceiver class

Log: Write ut for the trashfileeventreceiver class
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 committed:
fix: write ut

Write ut for the filecopymovejob class

Log: Write ut for the filecopymovejob class
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 committed:
feat: Write ut

Write ut for the fileoperationsservice class

Log: Write ut for the fileoperationsservice class
Task: https://pms.uniontech.com/task-view-257955.html

max-lvs reviewed (commented):
没的title， 内容可以写哪些领域的UT，就单单的义 write ut， 太简单了

liyigang1 commented:
> 没的title， 内容可以写哪些领域的UT，就单单的义 write ut， 太简单了

已添加

max-lvs reviewed (dismissed):
+2

liyigang1 committed:
feat: write ut

Write ut for the operationsstackproxy class

Log: Write ut for the operationsstackproxy class
Task: https://pms.uniontech.com/task-view-257955.html

liyigang1 commented:
/merage && rebase

liyigang1 commented:
/merge
